session.go

```diff
@@ -720,6 +721,11 @@ func (s *session) handleTCPPackage() error {
 				if netError, ok = perrors.Cause(err).(net.Error); ok && netError.Timeout() {
 					break
 				}
+				if perrors.Cause(err) == io.EOF {
+					err = nil
```

Comment:
`err = nil` is not needed

Reply:
@wongoo https://github.com/dubbogo/getty/blob/master/session.go#L654-L657 still log output the EOF error, so I reset `err` to `nil`.

Reply:
get it